.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,37 @@
+name: Feature Request
+description: Suggest an idea for this project
+title: "[FEATURE]"
+labels: ["enhancement"]
+
+body:
+  - type: textarea
+    id: related-problem
+    attributes:
+      label: "What kind of feature are you missing? Where do you notice a shortcoming of PTB?"
+      description: "A clear and concise description of what the problem is."
+      placeholder: "Example: I want to do X, but there is no way to do it."
+    validations:
+      required: true
+      
+  - type: textarea
+    id: solution
+    attributes:
+      label: "Describe the solution you'd like"
+      description: "A clear and concise description of what you want to happen."
+      placeholder: "Example: I think it would be nice if you would add feature Y so it will make it easier."
```

```suggestion
      placeholder: "Example: I think it would be nice if you would add feature Y so I can do X."
```

I think relating these to each other is a better experience

---

Done ✅